src/wfcopy.c

```diff
@@ -1699,10 +1699,11 @@ GetNextPair(PCOPYROOT pcr, LPTSTR pFrom,
                   goto ReturnPair;
                }
 
-			   //
-               // Directory: operation is recursive.
+			      //
+               // Directory: operation is recursive, but not for junctions and symlinks
                //
-               pcr->fRecurse = TRUE;
+               if (!(pDTA->fd.dwFileAttributes & (ATTR_SYMBOLIC | ATTR_JUNCTION)))
+                  pcr->fRecurse = TRUE;
                pcr->cDepth = 1;

```

Comment:
this code changes from always setting fRecurse to TRUE to sometimes setting it to TRUE.  Code just above sets that field to FALSE.  Thus I don't think we can be sure the field is FALSE and thus only have to conditionally set it to true.

So, seems like we should set fRecurse either way.  Make sense?

Reply:
Right. Now it is set correctly regardless of any precondition